requirements.txt

```diff
@@ -1,5 +1,5 @@
-PyJWT==1.1.0
-slumber==0.7.1
+# Package requirements
+-e .
```

Comment:
@jimabramson - In production, `pip install -r requirements.txt` would (should) never be run on this package.